CompilerForCAP/tst/loop_extraction.tst

```diff
@@ -50,8 +50,8 @@ end
 
 ## inspired by UniversalMorphismIntoDirectProductWithGivenDirectProduct in SkeletalFinSets
 gap> func :=  function ( cat_1, objects_1, T_1, tau_1, P_1 )
->     return ObjectifyMorphismWithSourceAndRangeForCAPWithAttributes(
->        rec( ), cat_1,
+>     return CreateCapCategoryMorphismWithAttributes(
+         cat_1,
```

Comment:
```suggestion
>        cat_1,
```

That's why the diff is looking weird. Interesting that its possible to interrupt the code without causing a syntax error :D

Reply:
Ugh, thank you. :)